packages/api/src/apps/api/app.ts

```diff
@@ -29,12 +29,12 @@ export default async (env: ProcessEnv) => {
   // we need web3 for syth price feeds
   const web3 = new Web3(env.CUSTOM_NODE_URL);
 
-  // how many blocks to skip before running updates on contract state
-  const updateBlocks = Number(env.UPDATE_BLOCKS || 1);
+  // how often to run expensive state updates, defaults to 10 minutes
+  const updateRateS = Number(env.UPDATE_RATE_S || 10 * 60);
   // default to 10 days worth of blocks
   const oldestBlock = Number(env.OLDEST_BLOCK_MS || 10 * 60 * 60 * 24 * 1000);
 
-  assert(updateBlocks > 0, "updateBlocks must be 1 or higher");
+  assert(updateRateS > 1, "UPDATE_RATE_S must be 1 or higher");
```

Comment:
This error message or condition seems to be a typo. This should either be >= 1 or it should say "must be greater than 1".

Reply:
good call